kloppy/infra/serializers/event/statsbomb/helpers.py

```diff
@@ -92,8 +98,8 @@ def get_player_from_freeze_frame(player_data, team, i):
             return team.get_player_by_id(player_data["player"]["id"])
         elif player_data.get("actor"):
             return event.player
-        elif player_data.get("keeper"):
-            return team.get_player_by_position(position_id=1)
+        # elif player_data.get("keeper"):
+        #     return team.get_player_by_position(position_id=1)
```

Comment:
I assume you will update this once the get_player_by_position method is updated to `return team.get_player_by_position(position_id=1, period_id=event.period_id, timestamp=event.timestamp)`?